docker-compose_production_bind.yml

```diff
@@ -289,6 +289,37 @@ services:
         target: /home/mig/state/mig_system_run
     command: /app/docker-entry.sh -k
 
+  migrid-lustre-quota:
+    container_name: migrid-lustre-quota
+    image: ucphhpc/migrid${CONTAINER_TAG}
+    network_mode: "host"
+    environment:
+      TZ: ${TZ}
+    depends_on:
+      - migrid
+    cap_add:
+      - ALL
+    volumes:
+      - type: bind
+        source: ${QUOTA_LUSTRE_BASE}
+        target: /lustre/${DOMAIN}
+      - type: bind
+        source: ${QUOTA_GOCRYPTFS_XRAY}
+        target: /usr/local/bin/gocryptfs-xray
+      - type: bind
+        source: ${QUOTA_GOCRYPTFS_SOCK}
+        target: /var/run/gocryptfs.${DOMAIN}.sock
+      - type: volume
+        source: mig
+        target: /home/mig/mig
+      - type: volume
+        source: state
+        target: /home/mig/state
+    command: /bin/bash -c \
```

Comment:
Minor suggestion but I think it would be more readable to drop the negation and use 
`[[ "${QUOTA_BACKEND}" =~ ^lustre.*$ ]] && miglustrequota.py -c /home/mig/mig/server/MiGserver.conf || exit 0` 

As exemplified in this simple test:
```
~$ QUOTA_BACKEND="lustre.bla"
~$ [[ "${QUOTA_BACKEND}" =~ ^lustre.*$ ]] && echo "run lustre quota" || echo "exit"
run lustre quota
~$ QUOTA_BACKEND=""
~$ [[ "${QUOTA_BACKEND}" =~ ^lustre.*$ ]] && echo "run lustre quota" || echo "exit"
exit
```